src/pudl/metadata/sources.py

```diff
@@ -607,6 +691,41 @@
         "license_raw": LICENSES["us-govt"],
         "license_pudl": LICENSES["cc-by-4.0"],
     },
+    "nrel_atb": {
+        "title": "NREL Annual Technology Baseline (ATB)",
```

Comment:
NREL publishes an ATB for both Electricity and Transportation.  It seems unlikely that we'll ever be working with the transportation data, but maybe it's worth noting that we're talking about the electricity data in the title and description.
```suggestion
        "title": "NREL Annual Technology Baseline (ATB) for Electricity",
```